crates/forge/src/lib.rs

```diff
@@ -216,21 +209,22 @@ fn to_runnable(
 /// * `runner_config` - A configuration of the test runner
 /// * `corelib_path` - Absolute path to the Cairo corelib
 /// * `contracts` - Map with names of contract used in tests and corresponding sierra and casm artifacts
-/// * `predeployed_contracts` - Absolute path to predeployed contracts used by starknet state e.g. account contracts
 ///
 
 #[allow(clippy::implicit_hasher)]
 pub async fn run(
-    package_path: &Utf8Path,
     package_name: &str,
-    package_source_dir_path: &Utf8Path,
+    snforge_target_dir_path: &Utf8Path,
```

Comment:
Why not just `target_dir_path`

Reply:
It's in the context of this crate anyways.

Reply:
Disagree, with `target_dir_path` you don't really know if it is a path to `target` or `target/dev/snforge`

Reply:
It's not immediately obvious that it is meant to be a path to `target/dev/snforge` with this name either. Target dir could also suggest a directory for which you want to run tests i.e. scarb package.

Reply:
I think it's obvious what target directory is considering we are in Rust-like environment